share/doc/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+if(NOT VOTCA_SPHINX_DIR)
+  set(VOTCA_SPHINX_DIR "${CMAKE_CURRENT_BINARY_DIR}")
+endif()
+
+file(GLOB NOTEBOOKS "${PROJECT_SOURCE_DIR}/GROMACS/Methane/QMMM_GROMACS.ipynb")
+list(APPEND NOTEBOOKS "${PROJECT_SOURCE_DIR}/LAMMPS/Thiophene/QMMM_LAMMPS.ipynb")
+list(APPEND NOTEBOOKS "${PROJECT_SOURCE_DIR}/tools/dftgwbse_CH4/DFTGWBSE_ENERGY.ipynb")
+list(APPEND NOTEBOOKS "${PROJECT_SOURCE_DIR}/tools/dftgwbse_CO_geoopt/DFTGWBSE_OPTIMIZATION.ipynb")
+# list(APPEND NOTEBOOKS "${PROJECT_SOURCE_DIR}/GROMACS/KMC_Methane/GROMACS_KMC.ipynb")
+list(APPEND NOTEBOOKS "${PROJECT_SOURCE_DIR}/LAMMPS/KMC_Thiophene/LAMMPS_KMC.ipynb")
+
+find_package(JUPYTER_NBCONVERT 5.4.1 REQUIRED)
```

Comment:
I think the previous solution was better now the build requires `nbconvert` even to build xtp without any documentation.

Reply:
I have made a `revert`. The problem is that sphinx needs `nbconvert` to create the documentation and there is not suitable `nbconvert` for ubuntu. So, can we skip the Sphinx build on ubuntu

Reply:
We only have to use it on one single platform. Despite all that, @felipeZ could you add sth. to the developer guide in votca/votca about what the documentation does?

Reply:
> I have made a `revert`. The problem is that sphinx needs `nbconvert` to create the documentation and there is not suitable `nbconvert` for ubuntu. So, can we skip the Sphinx build on ubuntu

Wouldn't
```
find_package(JUPYTER_NBCONVERT 5.4.1)
if(JUPYTER_NBCONVERT_FOUND AND SPHINX_FOUND)
```
take care of that?

Reply:
> We only have to use it on one single platform. Despite all that, @felipeZ could you add sth. to the developer guide in votca/votca about what the documentation does?

Correct. We can exclude stuff in the setup action, but I don't want the user to not being able to build xtp on Ubuntu because `JUPYTER_NBCONVERT` is required. We should try to find a new enough version and if that fails, we just skip that part of the documentation build.

Reply:
The problem is the following, there is a `index.rst` file in `VOTCA/VOTCA` that contains a reference to the `XTP-TUTORIAL.rst` file. When sphinx  is trying to build the documentation it will go first to `index.rst` then to `XTP-TUTORIAL.rst`. The problem is that inside that last file there are reference to the jupyter notebooks and sphinx will always try to convert those Jupyter notebooks into documentation by running them.
On the other hand,  the statement:
```cmake
find_package(JUPYTER_NBCONVERT 5.4.1)
if(JUPYTER_NBCONVERT_FOUND AND SPHINX_FOUND)
```
Can just try to run the notebooks or just copy them as they are, but Sphinx will always try to run them anyway and seems there is no  `nbconvert` tool it would fail.
So, I agree with @junghans  that the best alternative is just excluding the Sphinx build on Ubuntu.

Reply:
If we exclude it in the CI it will still trip up user on ubuntu running it manually.

Can we tell Sphinx to ignore the broken links? Or just create a mock version of xtp-tutorials.rst?

Reply:
We can ignore broken links, with `nbsphinx_allow_errors = True`  in the `config.py` file. I am going to do a PR